src/main/resources/application.yaml

```diff
@@ -91,10 +61,51 @@ feign:
   httpclient:
     enabled: true
 
+core_case_data:
+  api:
+    url: ${CORE_CASE_DATA_API_URL:http://localhost:4452}
+  caseTypeId: ${CORE_CASE_DATA_CASE_TYPE_ID:Benefit}
+  jurisdictionId: ${CORE_CASE_DATA_JURISDICTION_ID:SSCS}
+
+sscs:
+  serviceCode: ${SSCS_SERVICE_CODE:BBA3}
+
+judicial-ref:
```

Comment:
Have all these been added to their respective places in the Jenkinsfile and chart?

Reply:
I think this is sorted, there is a ticket for adding the secrets